.env.example

```diff
@@ -8,19 +8,15 @@ VUE_APP_ADDRESS_COMPLETE_KEY=
 VUE_APP_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
 VUE_APP_AUTH_WEB_URL="https://dev.account.bcregistry.gov.bc.ca/"
 VUE_APP_BUSINESSES_URL="https://dev.account.bcregistry.gov.bc.ca/"
-# for Legal Name feature branch only:
```

Comment:
This is legal name branch code, so I didn't see a reason to keep this comment or the main branch values below.